src/material/core/style/_focus-indicators.scss

```diff
@@ -0,0 +1,40 @@
+// Mixin that renders the focus indicator structural styles.
+@mixin private-structural-styling($prefix) {
+  position: relative;
+
+  &::before {
+    @include layout-common.fill();
+    box-sizing: border-box;
+    pointer-events: none;
+    border: var(
+        --#{$prefix}-focus-indicator-border-width,
+        $private-default-focus-indicator-border-width
+      )
+      var(--#{$prefix}-focus-indicator-border-style, $private-default-focus-indicator-border-style)
+      var(--#{$prefix}-focus-indicator-border-color, $private-default-focus-indicator-border-color);
+    border-radius: var(
+      --#{$prefix}-focus-indicator-border-radius,
+      $private-default-focus-indicator-border-radius
+    );
+  }
+}
+
+// Mixin that handles null values and defines CSS variables accordingly.
+@mixin private-create-variable($name, $value) {
```

Comment:
You can rename this one to `_create-variable` since it's only used within this file.